tests/lib/assembleurl.js

```diff
@@ -0,0 +1,22 @@
+// An expander to put together a base URL and relative path.
+// TODO: Consider moving this to a stringTemplate based system instead.
+"use strict";
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+
+fluid.registerNamespace("gpii.pouch.tests");
+gpii.pouch.tests.assembleUrl = function (baseUrl, path, that) {
```

Comment:
Linting still remains - also could you please add an "all-tests.js" driver at top level which runs the full test suite - Thanks